assets/blocks/view-certificate-block/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockVariation } from '@wordpress/blocks';
+
+const buttonVariationAttributes = {
+	url: '/course/sensei-view-certificate',
+	title: __( 'View Certificate', 'sensei-certificates' ),
+	text: __( 'View Certificate', 'sensei-certificates' ),
+	linkTarget: '_blank'
+};
+
+registerBlockVariation( 'core/button', {
+	name: 'view-certificate-button',
```

Comment:
We could add the prefix here in the name too.

Reply:
Fixed in cb0aa85bca5dbf5c231cddcbe16ab6cd7b0c84ad